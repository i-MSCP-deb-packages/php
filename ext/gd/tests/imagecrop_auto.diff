002+ int(99)
003+ int(99)
004+ Palette IMG_CROP_DEFAULT
005+ int(99)
006+ int(99)
002- int(11)
003- int(11)
004- Palette IMG_CROP_DEFAULT
005- int(11)
006- int(11)