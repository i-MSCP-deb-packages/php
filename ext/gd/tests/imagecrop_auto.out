TC IMG_CROP_DEFAULT
int(99)
int(99)
Palette IMG_CROP_DEFAULT
int(99)
int(99)
TC IMG_CROP_SIDES
int(11)
int(11)
Palette IMG_CROP_SIDES
int(11)
int(11)
TC IMG_CROP_BLACK
int(11)
int(11)
Palette IMG_CROP_BLACK
int(11)
int(11)
IMG_CROP_THRESHOLD
int(240)
int(134)